First_project/First_project/Search/SViewController.swift

```diff
@@ -70,9 +73,9 @@ class SViewController: UIViewController {
     }
 }
 
-extension SViewController: UISearchBarDelegate {
-    
-    func searchBarCancelButtonClicked(_ searchBar: UISearchBar) {
-        search(shouldShow: false)
-    }
-}
+//extension SViewController: UISearchBarDelegate {
```

Comment:
Better leave this extension here

Reply:
And also SViewController is not good naming. Make it self descriptive.
Same for FViewController